object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionTransfer.java

```diff
@@ -58,6 +73,16 @@
   public ObjectConstructionTransfer(final CFAnalysis analysis) {
     super(analysis);
     this.atypefactory = (ObjectConstructionAnnotatedTypeFactory) analysis.getTypeFactory();
+    ProcessingEnvironment env = atypeFactory.getChecker().getProcessingEnvironment();
+    treeBuilder = new TreeBuilder(env);
+  }
+
+  @Override
+  public TransferResult<CFValue, CFStore> visitTernaryExpression(
+      TernaryExpressionNode node, TransferInput<CFValue, CFStore> input) {
+    TransferResult<CFValue, CFStore> result = super.visitTernaryExpression(node, input);
```

Comment:
Not exactly related to this code, but did we add a test for calling `close()` on a ternary expression directly?

Reply:
https://github.com/kelloggm/object-construction-checker/pull/291/files#diff-d9c49afa35f8d4df342e326e80fd73fd3d5df0b9c0d3a1c95cd6662f84251d00R356